autoPyTorch/api/base_task.py

```diff
@@ -1245,17 +1239,32 @@ def __del__(self) -> None:
         # until the estimator is deleted
         self._backend.context.delete_directories(force=False)
 
-    @typing.no_type_check
     def get_incumbent_results(
-            self
-    ):
-        pass
+        self,
+        include_traditional: bool = False
+    ) -> Tuple[Configuration, Dict]:
+        """
+        Get Incumbent config and the corresponding results
+        Args:
+            include_traditional:
 
-    @typing.no_type_check
-    def get_incumbent_config(
-            self
-    ):
-        pass
+        Returns:
+
+        """
+        assert self.run_history is not None, "No Run History found, search has not been called."
+        assert not self.run_history.empty(), "Run History is empty."
```

Comment:
I think this makes more sense as a ValueError? so that users can catch it if they want. Probably with a message saying:
```
Something went wrong, smac was not able to fit any model?
```

Reply:
yeah agreed